source/scss/components/_quicksearch.scss

```diff
@@ -0,0 +1,53 @@
+.qs-with-background {
+  bottom: 150px;
+  margin-bottom: -100px;
+  position: relative;
+  @include media-breakpoint-down("md") {
+    bottom: 100px;
+    margin-bottom: -90px;
+  }
+  @include media-breakpoint-between("md", "lg") {
+    bottom: 250px;
+    margin-bottom: -200px;
+  }
+}
+
+.qs-h1 {
+  font-size: 3em;
```

Comment:
Do you want to add the typography values to the custom-variables?
Maybe you find some useful values there.
Also em does not use the html font-size, but the parent's font-size. Is this intended?